prac_10/wiki.py

```diff
@@ -0,0 +1,31 @@
+"""
+CP1404 Practical
+Wiki api test program
+Expected: 40m
+Actual: 45m
+"""
+
+import wikipedia
+
+
+def main():
+    """Takes a title or phrase input from the user until blank, and prints the respective wiki summary."""
+    search = input("Wikipedia title/phrase: ")
+    while search != '':
+        try:
+            wiki_page = wikipedia.page(search, auto_suggest=False)
+        except wikipedia.exceptions.DisambiguationError as e:
+            print(f"DisambiguationError: {e.options}")
+            wiki_page = None
+        except wikipedia.exceptions.PageError as e:
+            print(f"PageError: {e}")
+            wiki_page = None
+
+        if wiki_page:
```

Comment:
Would it be better to put this section into the try? You could then remove the need for the if statement and repeating ```wiki_page = none``` in every except, since if it doesn't error than it found a page.